### ISSUE TITLE
[bug][docs] Missing information about removal of `-e`, `-e:b` and `-e:h` and what to use instead

### ISSUE THREAD

**Issue by bartoszkosiorek** [conan-io/docs#3654]
With Conan 2 the  `-e`, `-e:b` and `-e:h` parameters was removed:
* https://docs.conan.io/2/reference/commands/install.html#conan-install
* https://docs.conan.io/1/reference/commands/consumer/install.html
* https://github.com/conan-io/conan/blob/59029095e86eacd402083fca7c308b06889f6301/conan/cli/args.py#L35

There is missing information in documentation about it (e.g. in migration guide: https://docs.conan.io/1/migrating_to_2.0/general.html#environment-variables) and how to replace it.

**Comment by memsharded** [comment]
Hi @BartoszKosiorek-TomTom 

Thanks for reporting this.

There are no plans to allow command line global environment variables, but to rely on system ones, like:
```bash
$ MYVAR=1 conan install ...
# or
$ set MYVAR=1 && conan install ...
```

In general, it would be discouraged to rely on env-vars for Conan related control (the recommended way to pass information to Conan is the new ``conf`` system, with command line ``-c`` and profile ``[conf]``, and ``global.conf``). For tools env-vars, the ``[buildenv]`` would be recommended in profiles.

**Comment by memsharded** [comment]
> Unfortanately conf is missing possibility of setting only for specific package:

Not really. The ``conf`` syntax also supports definition per package, both in CLI and in profiles with patterns and package references like: ``-c mypkg/*:<conf>=<conf_value>``
